sdcm/utils/common.py

```diff
@@ -495,7 +495,7 @@ def __str__(self):
         return ex_str
 
 
-def clean_cloud_instances(tags_dict):
+def clean_cloud_resources(tags_dict):
```

Comment:
please assert when tags dict doesn't have `TestId` or `RunByUser` keys
I am concerned when somebody will by mistake run this without tags or another tag like NodeType.
I think we must support only cleaning only when we have TestId/RunByUser with or without additional params
i.e. cleaning resources of one user or of single test id.

Reply:
@bentsi these checks are present in sct.py::clean_resources function. You want add the assertion to clean_cloud_resources function?

Reply:
since now `clean_cloud_resources` is called not only from `sct.py` it is better to put those checks in the `clean_cloud_resources` function

Reply:
Added check for TestId or RunByUser in dict, if not found, print error message and return.